aerogear-windows-otp/Totp.cs

```diff
@@ -6,18 +6,37 @@ namespace AeroGear.OTP
 {
     public class Totp
     {
+        public enum Digits
```

Comment:
@edewit, you are right, `DIGITS_POWER` and an integer class member `digits`, are those that used throughout the code to produce an OTP code with a desired number of digits. However, the `Digits` enumeration is a mean of conveniently define a range of permitted values. See, since we use an integer variable `binary` to calculate the base for the OTP code, we cannot allow OTP codes more than 10 digits long. The same goes for zero and negative numbers assigned to `digits`. Although, we could use a `uint` constructor parameter and introduce range checks and asserts, using a dedicated enumeration feels a lot more naturally.